cip/2.testable/CIP2015-06-24-call-procedures.adoc

```diff
@@ -102,65 +128,40 @@ This calls `myProc` for each record produced by the preceding `MATCH` clause wit
+The procedure call still has the same effect on the cardinality of the query execution in this case.
+
 The `YIELD` subclause is *always* omitted if the procedure returns only records with no result fields (i.e. is of type `()`).
 
 ==== Standalone calls
 
 Procedures may also be called standalone, i.e. without taking arguments from or combining their results with other parts of a larger query.
-In this case, the trailing `RETURN` clause may be omitted and all variable in scope are returned implicitly by the statement.
-Furthermore, we propose two optional simplifications that are available for standalone calls only.
-These simplifications are outlined in the following:
+In this case, the trailing `RETURN` clause is omitted and all fields projected in the `YIELD` subclause of the `CALL` statement are implicitly returned by the query.
+A further simplification allowed in the standalone form is to use `YIELD *` to denote that all fields returned by the procedure are to be returned by the query.
+The `YIELD *` form is _only_ allowed in the standalone form of `CALL`.
 
-===== Calling with implicitly passed arguments (parameters)
-
-Standalone calls may omit passing arguments explicitly.  In this case, all required procedure arguments are taken implicitly from statement parameters with the same name.
+If the `YIELD` subclause is omitted in the standalone form of `CALL`, the semantics are the same as in the _In-query_ form, i.e. the query will not project any fields into the result, but the query will still return as many (empty) rows as produced by the called procedure.
 
 Again consider a call to the procedure `myProc(name``::``STRING?``,``id``::``INTEGER?)``::`` (last``::``STRING?)`:
 
-.Standalone call to a procedure using implicitly passed arguments
-[source, cypher]
-----
-CALL myProc YIELD last AS lastLogin
-----
-
-This is the same as executing:
-
-.Standalone call to a procedure using explicitly passed arguments
+.Calling procedure as a standalone call statement
 [source, cypher]
 ----
-CALL myProc($name, {id}) YIELD last AS lastLogin
+CALL myProc("Donald", {id}) YIELD *
 ----
 
-===== Calling without `YIELD`
-
-Standalone calls that omit the `YIELD` subclause will always project out all result fields.
-
-Again consider a call to the procedure `myProc(name``::``STRING?``,``id``::``INTEGER?)``::`` (last``::``STRING?)`:
-
-.Standalone call to a procedure without `YIELD`
-[source, cypher]
-----
-CALL myProc("Donald", 12)
-----
+The above example would call `myProc` with the constant argument `"Donald"` for the `name` parameter, and use the query parameter `id` for the `id` parameter of the procedure.
+All the returned fields would then be bound and returned by the query.
 
-This is the same as executing:
+If we instead consider the procedure `listProcedures()``::`` (name``::``STRING?, arguments``::``STRING?, result``::``STRING?)`:
 
-.Equivalent in-query call for standalone call to a procedure without `YIELD`
+.Calling procedure standalone
```

Comment:
Is "Calling procedure standalone" meant to be "Calling procedure as a standalone call statement" here?